Admin/AbstractAdmin.php

```diff
@@ -572,22 +560,7 @@ public function getDataSourceIterator()
         $datagrid = $this->getDatagrid();
         $datagrid->buildPager();
 
-        $fields = array();
-
-        foreach ($this->getExportFields() as $key => $field) {
-            $label = $this->getTranslationLabel($field, 'export', 'label');
-            $transLabel = $this->trans($label);
-
-            // NEXT_MAJOR: Remove this hack, because all field labels will be translated with the major release
-            // No translation key exists
-            if ($transLabel == $label) {
-                $fields[$key] = $field;
-            } else {
-                $fields[$transLabel] = $field;
-            }
-        }
-
-        return $this->getModelManager()->getDataSourceIterator($datagrid, $fields);
+        return $this->getModelManager()->getDataSourceIterator($datagrid, $this->getExportFields());
```

Comment:
I think @core23 wrote the instructions. Might be fine, but please double-check together